### ISSUE TITLE
Ability to exclude specific folders from 'Unread Mail Indiactor'

### ISSUE THREAD

**Issue by mikescott** [joelekstrom/fastmate#77]
I have a lot of folders, and subfolders, and subsubfolders, etc. It would be a lot of work for me to create and maintain the list of 'Specific folders' to include.

Basically, my use case is I just want to exclude the 'Drafts' folder as this doesn't technically contain any unread messages.

**Comment by joelekstrom** [comment]
I agree, that would be useful. I see two ways of going about this - either I always exclude the drafts folder (as you say, it makes no sense for it to be included), or I add an exclusion filter to the "For all folders" setting.

I like the first approach since it's simpler in a sense, but then Fastmate needs to check for the drafts folder in all available languages, so it could break if Fastmail updates their translations. Will think a bit about this!

**Comment by mdbraber** [comment]
@joelekstrom I don't think you'd need to check for folder names, as a Drafts folder always has the class `v-MailboxSource--drafts`. Regarding the exclusion filter - it could be a simple 'exclude' checkbox after the specific folders text field, which would negate anything in the the text field.

**Comment by joelekstrom** [comment]
Added drafts exclusion, will be released with next version! Thanks for the suggestion @mikescott (and easy solution @mdbraber 🙏)
